lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -1029,12 +1033,16 @@ toApiNetworkParameters (NetworkParameters gp sp pp) toEpochInfo = do
             $ unDecentralizationLevel
             $ view #decentralizationLevel pp
         , desiredPoolNumber = view #desiredNumberOfStakePools pp
-        , minimumUtxoValue = Quantity
-            $ fromIntegral
-            $ unCoin
-            $ view #minimumUTxOvalue pp
+        , minimumUtxoValue = toApiCoin $ case (view #minimumUTxOvalue pp) of
+            MinimumUTxOValue c ->
+                c
+            MinimumUTxOValueCostPerWord _perWord ->
+                txOutputMinimumAdaQuantity txConstraints TokenMap.empty
```

Comment:
So in the alonzo case we here ask the TransactionLayer what the min ada quantity would be, i.e. representing `adaPerWord` as a `minUTxOValue` like we already are doing.